atomic_reactor/cli/task.py

```diff
@@ -71,7 +71,7 @@ def binary_container_postbuild(task_args: dict):
     """
     params = TaskParams.from_cli_args(task_args)
     task = BinaryPostBuildTask(params)
-    return task.execute()
+    return task.execute(init_build_dirs=True)
```

Comment:
we should move the initialization for `prebuild` here as well

Reply:
that unfortunately wouldn't work, because the platforms needed for initialization are not known at the start of prebuild yet, only after the check_and_set_platforms plugin finishes

maybe we should have a separate task between Clone and Prebuild which figures out the enabled platforms :thinking:

Reply:
but that sounds more like a future improvement than an immediate fix

Reply:
hmm, it'll be really helpful if tekton had a sort of `init` phase where we could do these steps needed to init tasks.

Reply:
agree with doing it in future